src/styles/card.scss

```
@@ -0,0 +1,24 @@
+.card {
+  display: block;
+  width: 198px;
```

use box-sizing: border-box;
```suggestion
  width: 200px;
```